devel/addon/samplewebhook/chart/templates/deployment.yaml

```diff
@@ -25,19 +25,18 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
+            - --secure-port=8443
             - --tls-cert-file=/tls/tls.crt
             - --tls-private-key-file=/tls/tls.key
           # These are explicitly set as they are not granted by default in OpenShift
           securityContext:
-            capabilities:
-              add:
-                - NET_BIND_SERVICE
+            runAsNonRoot: true
           env:
             - name: GROUP_NAME
               value: {{ .Values.groupName | quote }}
           ports:
             - name: https
-              containerPort: 443
+              containerPort: 8443
```

Comment:
Reconfigured the sample ACME webhook to listen on a non-privileged port and to run as a non-root user.